src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -137,7 +137,10 @@ - (nullable grpc_call *)unmanagedCallWithPath:(NSString *)path
                         completionQueue:queue];
 }
 
-- (NSData *)nullTerminatedDataWithString:(NSString *)string {
+- (NSData *)nullTerminatedDataWithString:(NSString *_Nullable)string {
```

Comment:
Also annotate return type with `_Nullable`?

Reply:
Per offline discussion, assert argument nonnull at usages of the method is better.